docs/standard/get-started.md

```diff
@@ -7,23 +7,27 @@ ms.date: 05/01/2017
 ms.technology: dotnet-standard
 ms.assetid: 81c07080-acdf-4aef-a66d-0ab52fab2c04
 ---
-# Get Started
+# Get started
 
-There are a number of ways to get started with .NET. Because .NET is a massive platform, there are multiple articles in this documentation which show how you can get started with .NET, each from a different perspective.
+There are a number of ways to get started with .NET. Because .NET is a massive platform, there are multiple articles in this documentation that can help you get started with .NET, each from a different perspective.
 
 ## Get started using .NET languages
 
-* The [C# Getting Started](../csharp/getting-started/index.md) articles and [C# Tutorials](../csharp/tutorials/index.md) provide a number of ways to get started in a C#-centric way.
+* The [C# getting started](../csharp/getting-started/index.md) articles and [C# tutorials](../csharp/tutorials/index.md) provide ways to get started in a C#-centric way.
 
-* The [F# Getting Started](../fsharp/get-started/index.md) tutorials provide three primary ways you can use F#: with Visual Studio, Visual Studio Code, or command-line tools.
+* The [F# getting started](../fsharp/get-started/index.md) tutorials cover the three primary ways you can use F#: with Visual Studio, Visual Studio Code, or command-line tools.
 
-* The [Visual Basic Getting Started](../visual-basic/getting-started/index.md) articles provide guides for using Visual Basic in Visual Studio.
+* The [Visual Basic getting started](../visual-basic/getting-started/index.md) articles provide guides for using Visual Basic in Visual Studio.
 
 ## Get started using .NET Core
 
-* [Getting Started with .NET Core](../core/get-started.md) provides an overview of articles which show how to get started with .NET Core on different operating systems and using different tools.
+* [Get started with .NET Core](../core/get-started.md) provides an overview of articles that show how to get started with .NET Core on different operating systems and using different tools.
 
-* The [.NET Core Tutorials](../core/tutorials/index.md) detail a number of ways you can get started with .NET Core using your operating system and tooling of choice.
+* The [.NET Core tutorials](../core/tutorials/index.md) detail ways you can get started with .NET Core using your operating system and tooling of choice.
+
+## Get started using .NET Standard
+
+* The [.NET Standard](net-standard.md) article provides a detailed overview of .NET Standard.
```

Comment:
Probably for getting started, it would be better to point to the tutorial at https://docs.microsoft.com/en-us/dotnet/core/tutorials/library-with-visual-studio